Makefile

```diff
@@ -0,0 +1,56 @@
+#  Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+#  with the License. A copy of the License is located at
+#
+#  http://aws.amazon.com/apache2.0/
+#
+#  or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+#  OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+#  and limitations under the License.
+
+
+# Verbose env tag
+# Usage: pass VERBOSE=1 to command line if need verbose mode.
+# > make pecos VERBOSE=1
+ifdef VERBOSE
+	V_FLAG=-v
+else
+	V_FLAG=
+endif
+
+
+# Style and type checks
+format: flake8 isort mypy
+
+flake8:
+	@echo "Checking flake8 on PECOS source files..."
+	python3 -m flake8 ${V_FLAG} --config .flake8 ./pecos
+	@echo "Checking flake8 on PECOS test files..."
+	python3 -m flake8 ${V_FLAG} --config .flake8 ./test
+
+isort:
+	@echo "Checking isort on PECOS source files..."
+	python3 -m black ${V_FLAG} --check --config .style_type.cfg --line-length=100 --diff ./pecos --exclude 'pecos/_version.py'
+	@echo "Checking isort on PECOS test files..."
+	python3 -m black ${V_FLAG} --check --config .style_type.cfg --line-length=100 --diff ./test
+
+mypy:
+	@echo "Checking mypy on PECOS source files..."
+	python3 -m mypy ${V_FLAG} --config-file .style_type.cfg -p pecos
+	@echo "Checking mypy on PECOS test files..."
+	python3 -m mypy ${V_FLAG} --config-file .style_type.cfg `find ./test/ -type f -name "*.py"`
+
+
+# Install and unit test
+libpecos:
+	python3 -m pip install --upgrade pip
+	pip install ${V_FLAG} --editable .
+
+unittest: libpecos
```

Comment:
We can use Phony target. See https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html.
So we can do something like `make test` instead of `make unittest`

Reply:
Sure, modified.